src/python.rs

```diff
@@ -57,6 +57,10 @@ impl<'p> Python<'p> {
     /// Because the output lifetime `'p` is not connected to any input parameter,
     /// care must be taken that the compiler infers an appropriate lifetime for `'p`
     /// when calling this function.
+    ///
+    /// # Safety
+    /// The lifetime `'p` must be shorter than the period you *assume* that you have GIL.
+    /// I.e., `Python<'static>` is always *really* unsafe.
```

Comment:
👍 nice. Not sure the comma is necessary.

```suggestion
    /// I.e. `Python<'static>` is always *really* unsafe.
```

Reply:
I didn't know but it looks a matter of Blitish/American English https://english.stackexchange.com/questions/6894/should-there-be-a-comma-after-i-e

Reply:
Heh that makes sense with me being British 😂